_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Helix minimum spacing standards require: 
+
+- Line spacing for Roboto Regular 16px is 24px.
+- Paragraph spacing is at least 36px. 
+- Use whitespace and proximity to make relationships between content more apparent.
+- Style headings to group content, reduce clutter, and make it easier to scan and understand.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "Little spacing and unclear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "More spacing and clear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Provide easily identifiable feedback
+
+Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Ensure instructions are easy to identify and suggestions to help fix issues whenever errors are returned must be provided. Present any important feedback that requires user action in a prominent style.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Failed text inputs
+
+There are many acceptable ways to make text inputs visually accessible. If you’re indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the error message, the border color turns red and doubles in thickness from 1px to 2px to provide distinguishable visual queues not reliant on color.
+
+It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error state.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Error state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg)
```

Comment:
grab the error indicator icon in the text inputs doc and add it in front of the error message in this example.  Even if we don't call it out in the text, it's also an example of using shape to indicate state.